cli/remotebazel/remotebazel.go

```diff
@@ -1273,3 +1298,12 @@ func getAPIKeyFromConfig() (string, error) {
 	}
 	return apiKey, nil
 }
+
+// isTaskMisconfigured returns whether a task failed to execute because of a
+// configuration error that will prevent the action from executing properly,
+// even if retried.
+func isTaskMisconfigured(err error) bool {
+	return status.IsInvalidArgumentError(err) ||
+		status.IsFailedPreconditionError(err) ||
+		status.IsUnauthenticatedError(err)
+}
```

Comment:
nit: leave a note both here and in `executor.go` to keep the 2 implementations in sync?

Reply:
Moved the logic to a shared until func